pkgs/by-name/re/regclient/package.nix

```diff
@@ -16,15 +17,15 @@ let
   ];
 in
 
-buildGoModule rec {
+buildGoModule (finalAttrs: {
   pname = "regclient";
   version = "0.9.2";
-  tag = "v${version}";
+  tag = "v${finalAttrs.version}";
```

Comment:
```suggestion
```
it is not really a standard thing to declare tag here

Reply:
Thank you, cleaned it up